.github/workflows/stable-snapshot-timestamp.yml

```diff
@@ -76,9 +72,9 @@ jobs:
               echo "found ceremony branch $branch"
               branch_date=$(echo "${branch}" | cut -d '/' -f3)
               days_diff=$(( ($(date -d "00:00" +%s) - $(date -d "${branch_date}" +%s)) / (24*3600) ))
-              if [[ "$days_diff" -lt 7 ]]; then
-                # Detected ceremony within 7 days of current date
-                echo "detected ceremony branch $branch within 7 days, stopping automated cron"
+              if [[ "$days_diff" -lt 5 ]]; then
```

Comment:
if you remove the schedule part of the cron, then why is this job needed?

Reply:
Reverting this back to a cron job.